apps/rush-lib/src/logic/DependencySpecifier.ts

```diff
@@ -32,6 +32,7 @@ export class DependencySpecifier {
    * directory - A local directory
    * remote - An HTTP url to a .tar.gz, .tar or .tgz file
    * alias - A package alias such as "npm:other-package@^1.2.3"
+   * workspace - A package specified using workspace protocol such as "workspace:^1.2.3"
    */
   public readonly specifierType: string;
```

Comment:
I don't disagree. I've left it this way because it's the way it's been with that many different types in the past, but if you'd prefer an enum I'm not opposed

Reply:
Let's make it an enum.